app/client.py

```diff
@@ -0,0 +1,56 @@
+# client.py
+
+from app.bucketlist import bluBucketList
+
+class bluClient(object):
+    def __init__(self, userName, userEmail, userPassword):
+        self.userName = userName
+        self.userEmail = userEmail
+        self.userPassword = userPassword
+        self.bL_list = {}
+        self.followedLists = []
+
+
+    def createBList(self, list_name, list_year, list_month, list_quote, set_active):
```

Comment:
I wanted to reduce the number of characters in the names used in the class methods. I will change it if it is against naming convention